lib/ardb/has_slug.rb

```diff
@@ -53,11 +53,12 @@ def reset_slug
 
       def ardb_has_slug_generate_slug
         slug_attr_value = self.send(self.class.ardb_has_slug_config[:attribute])
```

Comment:
@jcredding would be nice to have that `a` var on these two lines as well: https://github.com/redding/ardb/pull/72/files#diff-ec24ac0ed2933d9bc636957b64d1c69fR66